lib/tpm2_session.h

```diff
@@ -76,6 +76,15 @@ void tpm2_session_set_key(tpm2_session_data *data, TPMI_DH_OBJECT key);
  */
 void tpm2_session_set_nonce_caller(tpm2_session_data *data, TPM2B_NONCE *nonce);
 
+/**
+ * Gets the nonceNewer parameter.
```

Comment:
nonceNewer is the internal name. The caller might not know that. What is this nonce value for. Describe it more high level terms.

Retrieves the nonce value as returned via starting a new session. This nonce value can be used for blah blah blah.

Reply:
fixed